RecoEgamma/EgammaPhotonProducers/src/PhotonProducer.cc

```diff
@@ -359,6 +359,19 @@ void PhotonProducer::fillPhotonCollection(edm::Event& evt,
     float sigmaIetaIeta = sqrt(locCov[0]);
     float r9 =e3x3/(scRef->rawEnergy());
 
+    float full5x5_maxXtal =   noZS::EcalClusterTools::eMax( *(scRef->seed()), &(*hits) );
```

Comment:
So this code was copied directly from https://github.com/CmsHI/cmssw/blob/forest_CMSSW_7_5_3_patch1/RecoEgamma/EgammaPhotonProducers/src/GEDPhotonProducer.cc#L502 which also does not checkout for null so I didn't think about this too hard. 

I can add a non-null check to this file, but I'm loathe to change anything in GEDProducer to prevent PR creep.